packages/animations/example/android/app/build.gradle

```diff
@@ -37,12 +37,12 @@ android {
     }
 
     defaultConfig {
-        applicationId "dev.flutter.packages.animations.example"
```

Comment:
Here and elsewhere: we should keep the more specific application ids.

Reply:
Reverted this change, by re-running `flutter create --org= dev.flutter.packages.animations .`